common/config/rush/pnpm-lock.yaml

```diff
@@ -682,24 +779,24 @@ importers:
       '@itwin/itwinui-icons-react': 1.8.0_react-dom@17.0.2+react@17.0.2
       '@itwin/itwinui-react': 1.35.0_react-dom@17.0.2+react@17.0.2
     devDependencies:
-      '@itwin/appui-abstract': 3.2.4_@itwin+core-bentley@3.2.4
-      '@itwin/appui-layout-react': 3.2.4_d8b925a165b91dccf2814af5fd373d12
-      '@itwin/appui-react': 3.2.4_a110f0083bcb27abd0b8fad45f1a2e82
-      '@itwin/build-tools': 3.2.4
+      '@itwin/appui-abstract': 3.1.2_@itwin+core-bentley@3.2.4
```

Comment:
Versions got downgraded for some reason. Please investigate.

Reply:
Seems to be good now